CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ set(IDF_DEPS
     espcoredump
     hal
     heap
-    vfs)
+    vfs
+    mdns)
```

Comment:
we should have a note in the readme likely to ensure the consumer code has the required idf_component.yml dependency declared.